resources/microvm-kernel-arm64.config

```diff
@@ -193,7 +193,6 @@ CONFIG_SIGNALFD=y
 CONFIG_TIMERFD=y
```

Comment:
There is a huge difference in the kernel configs and zswap is just a small part.  Why all these changes? How did you generate it and from where you started?

Why do we need a different config with initramfs, I guess if you don't specify initramfs at startup it shouldn't add any penalty even if the config has **support** for it.

Reply:
Maybe I should have mentioned it, apart from removing ZSWAP, I also tried to bring the arm64 config as close to the x86_64 config. I mentioned this in the commit associated. One major difference between the 2 was that the arm64 config had a lot of loadable modules I.e (=m) while x86_64 did not have any and so I tried to make the 2 as similar as possible.

Reply:
I will try to see if using initrd enabled kernel will work even if no initrd is offered.

Reply:
Indeed, we do not need a different config with initramfs, @raduiliescu take another look.